index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!</h1>
+    <div id="main">
+      <h2 id="title">Mr. Samuel Bob</h2>
+      <div id="img-div" class="imageWrapper">
+        <figcaption id="img-caption">Tribute Image</figcaption>
+        <img id="image" src="img/image.jpg" />
+      </div>
+      <p id="tribute-info">Subject of the Tribute</p>
+      <a href="https://www.thegravesendflorist.co.uk/design-20179416/dummy-tribute.htm" id="tribute-link" target="_blank">Link to Tribute</a>
+    </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Kindly consider removing this comment from your code. Large comments like this makes your code untidy and so unprofessional.